### DISCUSSION TITLE
WSL doesn't let VMware or VBox work

### DISCUSSION THREAD

**Discussion by diegonzalezro** [microsoft/WSL#6213] [General] [answered]
After installing WSL and WSL2 my virtual machines stopped working. I uninstall WSL2  and WSL because maybe that was the issue but the problem persist.
How should I proceed to make my virtual machines work again?

**Comment by fuomag9** [comment]
Latest VMware release has support for HyperV as an hypervisor, as WSL forces it, I don't know about virtualbox but I believe there should be a setting to change to use HyperV as well

**Comment by ghost** [comment, marked as answer]
Are you using the latest version of VMware/VBox? See https://docs.microsoft.com/en-us/windows/wsl/wsl2-faq#will-i-be-able-to-run-wsl-2-and-other-3rd-party-virtualization-tools-such-as-vmware-or-virtualbox

**Reply by diegonzalezro** [reply]
Yeah, was the first thing I did. 

In the end I found that I also have to turn off more features on Windows features, and they started to work again.

In order to help other people with this problem these are the principle ones:
- Windows subsystem Linux
- HyperV
- Virtual machine platform
